app/ide-desktop/lib/dashboard/src/authentication/src/authentication/providers/session.tsx

```diff
@@ -100,6 +100,13 @@ export const SessionProvider = (props: SessionProviderProps) => {
          * https://github.com/aws-amplify/amplify-js/issues/3391#issuecomment-756473970 */
         window.history.replaceState({}, "", MAIN_PAGE_URL);
         refreshSession();
+        /** Typescript tells us we don't need the final condition because this is an exhaustive
```

Comment:
TypeScript has upper-case S in name.

Reply:
implementation has been changed to `switch` so this is no longer relevant